tsconfig.esm.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig",
+  "compilerOptions": {
+    "outDir": "./lib-esm",
+    "target": "ESNext",
```

Comment:
We should target one specific node version. I therefore recommend using https://github.com/tsconfig/bases

https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping